src/dfx/src/lib/operations/canister/motoko_playground.rs

```diff
@@ -65,12 +65,17 @@ impl TryFrom<InstallMode> for PlaygroundInstallMode {
         match m {
             InstallMode::Install => Ok(Self::Install),
             InstallMode::Reinstall => Ok(Self::Reinstall),
-            InstallMode::Upgrade {
-                skip_pre_upgrade: Some(false) | None,
-            } => Ok(Self::Upgrade),
-            InstallMode::Upgrade {
+            InstallMode::Upgrade(
+                Some(CanisterUpgradeOptions {
+                    skip_pre_upgrade: None | Some(false),
```

Comment:
Starting to wonder why skip_pre_upgrade isn't `bool` instead of `Option<bool>`

Reply:
No breaking changes 🥲